lib/basedriver/commands/images.js

```diff
@@ -34,28 +35,36 @@ const DEFAULT_MATCH_THRESHOLD = 0.4;
 commands.compareImages = async function compareImages (mode, firstImage, secondImage, options = {}) {
```

Comment:
Should we update the docstring for `options` argument?

Reply:
Also the description of the returned result must be updated

Reply:
As the existing comment says, it's heavily dependent on the mode and options (in appium-support).

Should we provide more detail specifically for MATCH_TEMPLATE_MODE?

Reply:
I assume we should. It would be very hard for clients to understand how to use the feature without a proper documentation. Not everyone could read javascript

Reply:
👌